src/component/foodcardlist/pantry/PantryStart.jsx

```diff
@@ -22,57 +23,90 @@ const data = [
   {
     "name": "Lemons",
     "image": lemonImg,
-    "selected": false
+    "select": false
   },
   {
     "name": "Cheese",
     "image": cheeseImg,
-    "selected": false
+    "select": false
   }
 ]
 
-
-
 const PantryStart = () => {
   const themeDesign = useTheme();
   const [ingredients, setIngredients] = useState(data);
-  const [selected, setSelected] = useState([])
+  const [select, setSelect] = useState([]);
   const [filter, setFilter] = useState('');
 
   const handleFilter = (e) => {
     setFilter(e.target.value.toLowerCase());
   }
 
+  const handleDelete = (e) => {
+    e.preventDefault();
+    let index = e.currentTarget.value;
+    let hold = [...ingredients];
+    let name = hold[index].name;
+
+    //if selected remove from select
+    // if(obj[name]) {
+    //   delete obj[name];
+    //   setSelect(obj);
+    // }
+    if (select.indexOf(name) !== -1) {
+      setSelect(select.filter(item => item !== name))
+    }
+
+    //remove from ingredients list and set new list
+    hold.splice(index, 1);
+    setIngredients([...hold]);
+    console.log(select);
+  }
+
   const handleSelect = (e) => {
-    console.log(e.target.name)
+    // console.log(e.currentTarget);
+    let name = e.target.name;
+    // let obj = select;
+
+
+    if (select.indexOf(name) === -1) {
+      // obj[name] = name;
+      // setSelect(obj);
+      setSelect([...select, name]);
+      console.log('add', select)
+    } else {
+      // delete obj[name]
+      // setSelect(obj);
+      setSelect(select.filter(item => item !== name));
+      console.log('delete', select);
+    }
   }
 
   const handleSubmit = (e) => {
```

Comment:
is this just for dev purposes? The user generally won't be looking at the console.